pylint/checkers/exceptions.py

```diff
@@ -69,7 +69,7 @@ def _is_raising(body: list) -> bool:
     "E0702": (
         "Raising %s while only classes or instances are allowed",
         "raising-bad-type",
-        "Used when something which is neither a class nor an instance "
+        "Used when something which is neither a class, an instance nor a string "
```

Comment:
I don't understand this change. I don't think you can raise a string ?
```
>>> raise "A string"
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: exceptions must derive from BaseException
```

Reply:
This addition comes from [pylint-errors E0702](https://vald-phoenix.github.io/pylint-errors/plerr/errors/exceptions/E0702). Should it be reverted?

Reply:
Yeap I think it's wrong.
```suggestion
        "Used when something which is neither a class nor an instance "
```